README.md

```diff
@@ -59,27 +57,26 @@ services:
     ports:
       - 8545:8545
 ```
+The ganache image 
+comes with 10 prefunded ethereum accounts  you can use for testing. These are the usual ganache test accounts. They have hold some ETH and GEN.
 
 
-## What you need
-
-- `npm`
-- `docker`
-
-## Create a new release
+## How to create and publish new docker images
 
-`./release.sh` will create a new release, which will run the following steps:
+`./release.sh` will create a rebuild the images and publish them on dockerlhub.
 
 1. (re)start fresh docker containers for ipfs, postgres, graph-node, ganache
 1. deploy the contracts, DAOs, proposals, etc to ganache using `npm run deployEthereum`
 1. build a subgraph that indexes these using `npm run deploySubgraph`
 1. tag and publish the docker contains to dockerhub
 
 
-`./release.sh -d` will run the script in development mode, which will run all the steps except publish the ersult on docker hub
+`./release.sh -d` will run the script in development mode, which will run all the steps except publish the result on docker hub. This is useful for testing
 
 
-## Instructions
+
+## How to create a test_env release for a new `arc` or `subgraph` package 
```

Comment:
Is there any other scenario than this in which we would need to run release.sh or do anything else than this?